test/runtests.jl

```diff
@@ -16,3 +18,43 @@ end
     @test a ≈ result[1] atol=0.05
     @test b ≈ result[2] atol=0.05
 end
+
+@testset "ranefinfo" begin
+    m1 = fit(
+        MixedModel,
+        @formula(1000/reaction ~ 1 + days + (1+days|subj)),
+        MixedModels.dataset(:sleepstudy),
+    )
+    reinfo = ranefinfo(m1)
+    @test isone(length(reinfo))
+    @test keys(reinfo) == (:subj, )
+    re1 = first(reinfo)
+    @test isa(re1, RanefInfo)
+    @test re1.cnames == ["(Intercept)", "days"]
+    @test first(re1.levels) == "S308"
+    @test first(re1.ranef) ≈ -0.081830283935302 atol=1.0e-5
+    @test first(re1.stddev) ≈ 0.14035486016644683 atol=1.0e-5
+    f = Figure();
+    @test f.content == Any[]
+    caterpillar!(f, re1);
+    @test length(f.content) == 2
+    @test isa(first(f.content), Axis)
+    @test isone(f.layout.nrows)
+    @test f.layout.ncols == 2
+end
+
+@testset "shrinkageplot" begin
+    m1 = fit(
+        MixedModel,
+        @formula(1000/reaction ~ 1 + days + (1+days|subj)),
+        MixedModels.dataset(:sleepstudy),
+    )
+    f = shrinkageplot(m1)
+    @test isone(length(f.content))
+end
+
+@testset "utilities" begin
+    ppts = MixedModelsMakie.ppoints(64)
+    @test length(ppts) == 64
+    @test first(ppts) ≈ inv(128)
```

Comment:
I was confused by this for a second until I realized that everything is set up to work on the *open* interval.

Reply:
Yes, the idea is to create `n` equally spaced points on the open interval as the midpoints of the `n` equal-width subintervals of (0,1).